File: Mobile/app/src/main/java/com/example/realestateclient/data/models/ViewingRequest.kt
package com.example.realestateclient.data.models

data class ViewingRequest(
    val id: Int = 0,
    val propertyId: Int,
    val preferredDate: String,
    val preferredTime: String,
    val comment: String?,
    val status: String? = null,
    val propertyAddress: String? = null
)

File: Desktop/RealEstateAgency/RealEstateAgency/Models/ViewingRequest.cs
using System;

public class ViewingRequest
{
    public int Id { get; set; }
    public int ClientId { get; set; }
    public int PropertyId { get; set; }
    public DateTime RequestDate { get; set; }
    public string PreferredTime { get; set; }
    public string Status { get; set; }
    public string Comment { get; set; }

    public string ClientName { get; set; }
    public string PropertyAddress { get; set; }
}